### DISCUSSION TITLE
Hidden files

### DISCUSSION THREAD

**Discussion by davimmt** [yassinebridi/serpl#21] [Q&A]
Currently, is there support for targetting hidden files in the directory? Also, ignore some paths within the root directory?

**Comment by yassinebridi** [comment]
There is no support for hidden files, i will try to add this as soon as possible.
as for ignoring some paths, i'm not sure what the best route to take here, is it to:
- Ignore paths per folder, meaning whenever you run `serpl` inside a folder, you would have to ignore some paths.
- ignore global folders, configured in `serpl`'s config file

**Reply by davimmt** [reply]
I don't know what the proposal of the tool is, however as I'm more familiar with "core" GNU/Linux utils, I thought of [grep](https://man7.org/linux/man-pages/man1/grep.1.html) `--exclude` options it's the most common among all. But again, I don't know your design patterns.

**Reply by yassinebridi** [reply]
I got you, i will think about what's the best user experience here.
Thanks for your input 🙏
